libvcx/src/disclosed_proof.rs

```diff
@@ -74,24 +74,25 @@ pub fn get_state(handle: u32) -> VcxResult<u32> {
 pub fn update_state(handle: u32, message: Option<&str>, connection_handle: u32) -> VcxResult<u32> {
     HANDLE_MAP.get_mut(handle, |proof| {
         trace!("disclosed_proof::update_state >>> connection_handle: {:?}, message: {:?}", connection_handle, message);
-
         if !proof.has_transitions() {
             trace!("disclosed_proof::update_state >> found no available transition");
             return Ok(proof.state());
         }
+        let send_message = connection::send_message_closure(connection_handle)?;
 
         if let Some(message_) = message {
```

Comment:
Personally, I would shy away from suffixing with `_` here (I don't like introducing new variable names when not necessary)

Reply:
updateed